draft-ietf-taps-transport-security.md

```diff
@@ -122,15 +122,16 @@ Services and features provided by transport protocols have been cataloged in {{?
 document supplements that work by surveying commonly used and notable network security protocols,
 and identifying the interfaces between these protocols and both transport protocols and
 applications.  It examines Transport Layer Security (TLS), Datagram Transport Layer Security (DTLS),
-IETF QUIC, Google QUIC (gQUIC), tcpcrypt, Internet Protocol Security (IPsec), SRTP (with DTLS),
+IETF QUIC, Google QUIC (gQUIC), tcpcrypt, Internet Protocol Security (IPsec), Secure Real-time Transport Protocol (SRTP) with DTLS,
 WireGuard, CurveCP, and MinimaLT. For each protocol, this document provides a brief description.
 Then, it describes the interfaces between these protocols and transports in {{transport-interface}}
 and the interfaces between these protocols and applications in {{application-interface}}.
 
-Selected protocols represent a superset of functionality and features a Transport Services system may
+A Transport Services system exposes an interface to different Transport Services to applications. Further, a Transport Services system maps the Transport Service that an application has requested to a specific protocol stack, including transport protocols and security protocols.
```

Comment:
```suggestion
A Transport Services system exposes an interface for applications to access various transport protocol features, including transports that offer security.
```

Reply:
How about the following?

"A Transport Services system exposes an application interface for accessing various (secure) transport protocol features."

Reply:
Thanks for the suggestions! I like both and I'm going with the latter.